### PR TITLE
Update requirements for v3.8

### PR DESCRIPTION
**Pull request by plettich**
Update requirements to the latest working versions for the v3.8
dev-cycle.

Working on #3111

### CODE REVIEW

**Comment by cornelinux** [comment]
@plettich could the failing test be related to your dependency updates?

File: requirements.txt [hunk 1]
```diff
@@ -31,67 +44,80 @@ Flask-Script==2.0.6
...
-idna==3.2; python_version > '3.5'
-importlib_metadata==2.1.1; python_version <= '3.5'
-importlib_metadata==4.6.0; python_version > '3.5'
+idna==3.3; python_version > '3.5'
+importlib-metadata==2.1.3; python_version <= '3.5'
+importlib-metadata==4.8.3; python_version == '3.6'
+importlib-metadata==4.11.3; python_version > '3.6'
 importlib-resources==3.3.1; python_version < '3.0'
 importlib-resources==3.2.1; python_version == '3.5'
+importlib-resources==5.4.0; python_version == '3.6'
```
Comment (cornelinux) [inline requirements.txt:57]: Hm, which importlib-resources is used for >3.6?

File: requirements.txt [hunk 2]
```diff
@@ -31,67 +44,80 @@ Flask-Script==2.0.6
...
 pyrad==2.4
-python-dateutil==2.8.1
-python-editor==1.0.4
-python-gnupg==0.4.7
-pytz==2021.1
+python-dateutil==2.8.2
+python-editor==1.0.4; python_version <= '3.5'
+python-gnupg==0.4.8
+pytz==2022.1
+PyYAML==5.4.1; python_version < '3.0'
```
Comment (cornelinux) [inline requirements.txt:94]: How crappy is 3.5?
Why do older versions of python allow a higher pyYAML version?

File: requirements.txt [hunk 3]
```diff
@@ -31,67 +44,80 @@ Flask-Script==2.0.6
...
+PyYAML==6.0; python_version > '3.5'
 qrcode==6.1; python_version <= '3.5'
-qrcode==7.1; python_version > '3.5'
-redis==3.5.3
+qrcode==7.3.1; python_version > '3.5'
+redis==3.5.3; python_version <= '3.5'
+redis==4.2.2; python_version > '3.5'
 requests==2.25.1; python_version == '3.5'
-requests==2.26.0; python_version != '3.5'
+requests==2.27.1; python_version != '3.5'
```
Comment (cornelinux) [inline requirements.txt:102]: Again, only Python 3.5 requires an older requests version?

**Review by cornelinux — APPROVED** [review]

File: requirements.txt (hunk 1 above)
Comment (plettich) [reply requirements.txt:57]: None, it is not used with Python > 3.6

**Pull request merged**